File: 5.py
print("Podaj rodzaj rownania \n(1) Liniowe \n(2) Kwadratowe\n")
i = input()
if i==1:
    print("Podaj wspolczynnik a: ")
    a = input()
    print("\nPodaj wspolczynnik b: ")
    b = input()
    print("\nPodaj wspolczynnik x: ")
    x = input()
    y = a*x + b
    print("\nWynik to: " + y)

elif i==2:
    print("Podaj wspolczynnik a: ")
    a = input()
    print("\nPodaj wspolczynnik b: ")
    b = input()
    print("\nPodaj wspolczynnik c: ")
    c = input()
    print("\nPodaj wspolczynnik x: ")
    x = input()
    y = a*(x**2) + b * x + c
    print("\nWynik to: " + y)

File: 5.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    int i;
    cout <<
    "Podaj rodzaj rownania" << endl <<
    "(1) Liniowe" << endl <<
    "(2) Kwadratowe" << endl;
    cin>>i;
    if(i==1)
    {
        int a,b,x,y;
        cout << "Podaj wspolczynnik a: ";
        cin>>a;
        cout << endl << "Podaj wspolczynnik b: ";
        cin>>b;
        cout << endl << "Podaj wspolczynnik x: ";
        cin>>x;
        y = a*x + b;
        cout << endl << "Wynik to: " << y;
    }
    else if(i==2)
    {
        int a,b,c,x,y;
        cout << "Podaj wspolczynnik a: ";
        cin>>a;
        cout << endl << "Podaj wspolczynnik b: ";
        cin>>b;
        cout << endl << "Podaj wspolczynnik c: ";
        cin>>c;
        cout << endl << "Podaj wspolczynnik x: ";
        cin>>x;
        y = a*pow(x,2) + b*x + c;
        cout << endl << "Wynik to: " << y;
    }
}
